specificatie/openapi.yaml

```diff
@@ -207,17 +207,28 @@ paths:
           $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/responses/default'
 
   /adresseerbareobjecten:
+    #334
     get:
       security:
         - apiKeyBAG: []
       operationId: raadpleegAdresseerbareobjecten
       tags:
         - Adresseerbaar object
       summary: "vindt verblijfsobjecten, ligplaatsen, standplaatsen"
-      description: "Zoek actuele adresseerbare objecten (verblijfsobjecten, standplaatsen of ligplaatsen) met een nummeraanduidingidentificatie of pandidentificatie. Gebruik de fields-parameter als je alleen specifieke velden in het antwoord wil zien, zie [functionele specificaties fields-parameter](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.2.0/features/fields.feature). Gebruik de expand-parameter als je het antwoord wil uitbreiden met (delen van) de gerelateerde resources adres en pand, zie [functionele specificaties expand-parameter](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.2.0/features/expand.feature)."
+      description: >-
+        "Zoek actuele adresseerbare objecten (verblijfsobjecten, standplaatsen of ligplaatsen) met een nummeraanduidingidentificatie, 
+        pandidentificatie, of met een status geconstateerd in combinatie met woonplaats identificatie, aangevuld met type en één of meer gebruiksdoelen. 
+        Gebruik de fields-parameter als je alleen specifieke velden in het antwoord wil zien, zie 
+        [functionele specificaties fields-parameter](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.2.0/features/fields.feature). 
+        Gebruik de expand-parameter als je het antwoord wil uitbreiden met (delen van) de gerelateerde resources adres en pand, zie 
+        [functionele specificaties expand-parameter](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.2.0/features/expand.feature)."
       parameters:
         - $ref: '#/components/parameters/nummeraanduidingIdentificatie-query'
         - $ref: '#/components/parameters/pandIdentificatie-query'
+        - $ref: '#/components/parameters/geconstateerd'
+        - $ref: '#/components/parameters/woonplaatsIdentificatie-query'
+        - $ref: '#/components/parameters/type'
+        - $ref: '#/components/parameters/gebruiksdoelen'
```

Comment:
Er kan meerdere gebruiksdoelen worden opgegeven. Welk operator wordt in dat geval gehanteerd? Willen we hier niet dezelfde flexibiliteit als bij het zoeken met een contour, met de array operator?

Reply:
Er kan een lijst worden opgegeven en als één van de gebruiksdoelen van een VBO in de lijst voorkomt dan wordt die VBO teruggegeven.

Reply:
@MelvLee
Denk je dat de oplossing die ik hierboven beschrijf voldoende flexibiliteit biedt?

Reply:
Ik denk dat het grootste deel van de gevraagde functionaliteit hiermee kan worden ingevuld.